kiosk/bin/kiosk-browser

```diff
@@ -21,13 +21,21 @@ parser.add_argument(
     'Keyboard combination to toggle between primary and secondary websites. (Default: "%s")'
     % default_toggle_key)
 
+parser.add_argument('system_name', help='System name displayed in user agent')
+parser.add_argument('system_version', help='System version displayed in user agent')
```

Comment:
What's the motivation of splitting into system name and system version?

It allows to fall back to `<name>/<version> <default-user-agent>` in case parsing fails, but it also complicates the implementation, and even forces specifying something for both name and version.

Could the caller decide which system identifier to use?